interface/reports/morphineReport.php

```diff
@@ -0,0 +1,164 @@
+<?php
+/**
+ * Morphine Report
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2016-2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+
+require_once ("../globals.php");
+require_once ($GLOBALS['srcdir'].'/patient.inc');
+use OpenEMR\Core\Header;
+
+$group = "t";
+$reportDataTotalPatients = morphineReport($group);
+
+$blurb1 = xlt('Number of Patients in this group are ');
+$blurb2 = xlt("Percentage of patients in this group ");
+$buttonLabel = xlt('View Patients'); 
+?>
+
+<html>
+<head>
+    <title>Morphine Report</title>
+
+    <?php Header::setupHeader(); ?>
+
+    <link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'] ?>/bootstrap-3-3-4/dist/css/bootstrap.min.css"> 
+    <script src="<?php echo $GLOBALS['assets_static_relative'] ?>/bootstrap-3-3-4/dist/js/bootstrap.min.js"></script>
```

Comment:
above 2 lines can be removed. these are already included in above Header::setupHeader call